### ISSUE TITLE
Don't error quit if project setup dialog has invalid resources

### ISSUE THREAD

**Issue by ioan-chera** [ioan-chera/eureka-editor#20]
At least two cases identified so far:

1. Invalid resource files (other than ugh)
2. Ugh files with syntax errors

Instead of suddenly terminating Eureka, they should calmly inform the user and (depending on user experience) either request correction or skip them and close the dialog anyway.

# Blocked by

- [x] #23 because we need a more robust wad system.
- [x] #97 because we need to easily handle lifetimes of failed objects
- [x] Scenario: create new project, prepare to overwrite another file. Give it a junk resource file. The operation will fail, but the old file will still be forever lost.

**Comment by ioan-chera** [comment]
Solved most of the problems for now. Will open a new issue upon new discoveries.
